grails-app/conf/Config.groovy

```diff
@@ -69,7 +69,7 @@ environments {
 }
 
 // See WebpackTagLib.groovy
-webpack.bundle.dir = '/webpack-bundles'
+webpack.bundle.dir = System.getProperty("streamr.webpack.bundle.location") ?: '/webpack-bundles'
```

Comment:
Is `streamr.webpack.bundle.location` a good parameter name?

Reply:
Yeah good name. Should rename `webpack.bundle.dir` to `streamr.webpack.bundle.location` in code for consistency.